### PR TITLE
OKTA-736503 - Minor updates to vanilla spa app

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Minor updates to this guide: https://developer.okta.com/docs/guides/auth-js-redirect/main/ including updating auth.js version and adding full code for ease of implementation.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-736503](https://oktainc.atlassian.net/browse/OKTA-736503)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/de77897a0973c32bdd88c11e9e0bb25654c73b16/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauth-js-redirect%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5fd7f5c0-afdc-45a1-bba9-e44b5341237e) | 56  | 71  | 95  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a8af54fa-e8e0-42da-848f-22de5b187009?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/fb8ff0b2a4428180c89bd97aed376b5d8d187125/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauth-js-redirect%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c1d6dfc8-93fe-4874-a4aa-e40395f41913) | 56  | 71  | 95  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b39d1dd3-24a6-42c5-8f1d-58d170647fb7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The integration includes configuration information required by the app to access
 
 1. [Sign in to your Okta organization](https://developer.okta.com/login) with your administrator account.
 1. Click **Admin** in the upper-right corner of the page.
-1. Open the Applications configuration pane by selecting **Applications** > **Applications**.
+1. Open the **Applications** configuration pane by selecting **Applications** > **Applications**.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:51]:
```suggestion
1. Go to **Applications** > **Applications**.
```

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ The integration includes configuration information required by the app to access
...
 1. Click **Create App Integration**, and then select a **Sign-in method** of **OIDC - OpenID Connect**.
 1. Select an **Application type** of **Single-Page Application**, then click **Next**.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:53]:
```suggestion
1. Select an **Application type** of **Single-Page Application**, and then click **Next**.
```

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 3]
```diff
@@ -70,7 +70,9 @@ To review or set trusted origins go to **Security** > **API** and select the **T
 
 ## Create a basic app
 
-To make this sample as versatile as possible, the following starter app redirects to Okta to sign in as you load it into the browser. In your own apps, you might want to initiate the redirect using a sign-in button. You might want to initiate the redirect when visiting a certain route that requires authentication (such as an admin page). The key is that you initiate the sign-in flow through the redirect.
+To make this sample as versatile as possible, the following starter app redirects to Okta to sign in as you load it into the browser. In your own apps, you might want to initiate the redirect using a button for sign-in. You might want to initiate the redirect when visiting a certain route that requires authentication (such as an admin page). The key is that you initiate the sign-in flow through the redirect.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:73]:
```suggestion
To make this sample as versatile as possible, the following starter app redirects to Okta to sign in as you load it into the browser. In your own apps, you might want to initiate the redirect using a button for sign-in. Or, you might want to initiate the redirect when visiting a certain route that requires authentication (such as an admin page). The key is that you initiate the sign-in flow through the redirect.
```

File: packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md [hunk 4]
```diff
@@ -192,7 +261,7 @@ If your app isn't functional, ensure that:
 * Your client ID is accurate from your Okta app integration.
 * Your `redirectUri` is accurate or the port number for your local web server is correct.
 * You've enabled a Trusted Origin for `http://localhost:9000`. See [About Trusted Origins](#about-trusted-origins).
-* If your app is bypassing the Okta Sign-In Widget, your user is already signed in. Use a new private or incognito browser window or optionally set the Authentication Policy for your app to always sign in. That is, **Re-authentication frequency is** set to **Every sign-in attempt**.
+* If your app is bypassing the Okta Sign-In Widget, your user is already signed in. Use a new private or incognito browser window or optionally set the Authentication Policy for your app to always sign in. That is, the **Re-authentication frequency is** set to **Every sign-in attempt**.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md:264]:
```suggestion
* If your app is bypassing the Okta Sign-In Widget, your user is already signed in. Use a new private or incognito browser window or optionally set the authentication policy for your app to always sign in. That is, the **Re-authentication frequency is** set to **Every sign-in attempt**.
```

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/94e0f318262479492fd520ac77eaa3a4081d6260/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauth-js-redirect%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2c004425-1c87-487c-92a3-eff446df9b5f) | 56  | 71  | 95  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c938bf30-0fd5-4741-af61-547b9bf46239?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/b4b2eba14617443fc26b7e75f44401669ab31dd3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauth-js-redirect%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ba47a053-69cc-4631-84dc-e1e2d6babf80) | 56  | 71  | 95  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.2fa5980e-72fe-4bb8-ba0d-190f7a1f4073?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/921c47ff3517ab9be057a372419de474ee65172d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauth-js-redirect%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8b5ca62a-364b-4dae-aa72-77847f7cea18) | 56  | 71  | 95  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4acd48f2-dcdf-456a-9cb6-323240f46ce6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/23c83ae4ed4540e7bbbdc3116df0fb9ef815e8f7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauth-js-redirect%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6cee4654-e072-4754-adea-a2be242e7e4d) | 56  | 71  | 95  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4a87a6d1-eb93-40a8-afba-fa66ef97d84d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**